### PR TITLE
Adding Jan 2020 Part 1 presentation link

### PR DESCRIPTION
**Pull request by ahmad-shahid**

### CODE REVIEW

File: strategy/dates.md [hunk 1]
```diff
@@ -10,6 +10,16 @@ Our next scheduled event is planned to be a two-part session on APIs ([Topic: Cr
 
 ## Past Events
 
+### January 2020 (Part 1)
```
Comment (s-laugh) [inline strategy/dates.md]: Let's keep this consistent... Maybe add a "part 1" in the link of the slides.
```suggestion
### January 2020
```

File: strategy/dates.md [hunk 2]
```diff
@@ -10,6 +10,16 @@ Our next scheduled event is planned to be a two-part session on APIs ([Topic: Cr
...
+
+**Topic:** [Creating & Using APIs from the store (using REST) (#34)](https://github.com/esdc-devcop/esdc-devcop.github.io/issues/34)  
+**Presentation:** [View the presentation](https://github.com/esdc-devcop/presentations/blob/master/Managing%20APIs%20and%20Microservices.pptx)  
+**Recording:** [Presentation Recording](https://gts-ee.webex.com/gts-ee/ldr.php?RCID=5d5c2e2cddb3f97316bbfa6618b4d270)  
+
+#### Resources - January 2020
```
Comment (s-laugh) [inline strategy/dates.md]: Also consistancy
```suggestion
#### Resources - Jan 2020
```

File: strategy/dates.md (hunk 1 above)
Comment (ahmad-shahid) [reply strategy/dates.md]: Fixed

File: strategy/dates.md (hunk 2 above)
Comment (ahmad-shahid) [reply strategy/dates.md]: Fixed

**Pull request merged**